quantized comparators benchmarking

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29437 quantized comparators benchmarking**

Differential Revision: [D18389909](https://our.internmc.facebook.com/intern/diff/D18389909)

Commit:
quantized comparators benchmarking

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qcomparators_test.py:
```diff
@@ -0,0 +1,66 @@
+        self.qop = op_func
+
+    def forward(self):
+        if self.out is not None:
```
Can you move the if statement to the init function? Just want to remove this extra branch prediction overhead from the reported time.

z-a-f commented on benchmarks/operator_benchmark/pt/qcomparators_test.py:
```diff
@@ -0,0 +1,66 @@
+                                                   zero_point=zero_point,
+                                                   dtype=dtype)
+        if other_scalar:
+            self.q_input_b = 42
```
It's the answer to everything: https://www.dictionary.com/e/slang/42/

Commit:
Update on "quantized comparators benchmarking"


Differential Revision: [D18389909](https://our.internmc.facebook.com/intern/diff/D18389909)

mingzhe09088 reviewed (approved):
LGTM!
